README.md

```diff
@@ -1,3 +1,190 @@
-# Codex Protocol | Codex Registry API
+# Codex Protocol | Codex Registry API _(service.codex-registry-api)_
 
-Documentation is coming soon! Feel free to [reach out on Telegram](https://t.me/codexprotocol) in the meantime.
+[![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
+
+> Node API for Codex Registry
+
+This is the Node API that serves as the off-chain backend for the [Codex Viewer](https://github.com/codex-protocol/web.codex-viewer). Among various other tasks, this API handles user authentication via [JSON Web Tokens](https://www.npmjs.com/package/jsonwebtoken), processes blockchain events via [EEL](https://github.com/codex-protocol/service.eel) & [Agenda](https://www.npmjs.com/package/agenda), manages MongoDB interactions via [Mongoose](https://www.npmjs.com/package/mongoose), and manages socket connections from the front end via [socket.io](https://www.npmjs.com/package/socket.io) & [Redis](https://www.npmjs.com/package/redis).
+
+## Table of Contents
+
+- [Security](#security)
+- [Background](#background)
+- [Install](#install)
+- [Usage](#usage)
+- [API](#api)
+- [Maintainers](#maintainers)
+- [Contribute](#contribute)
+- [License](#license)
+
+
+## Security
+
+It should be noted that this project requires the use of some "secret" variables that should be tailored to your development environment and never committed to the repository. These variables are read by [dotenv](https://www.npmjs.com/package/dotenv) and should be defined in a file called `.env` in the project root.
+
+After cloning the repository, you should duplicate and rename `.env.example` to `.env`, then tailor the variables to your local environment accordingly:
+
+```bash
+$ cp .env.example .env
+$ vi .env
+```
+
```

Comment:
I'm not sure we need this example. It's been made clear in the previous sentence what needs to be done. Also, this code snippet assumes people know how to use `vi` 😃

Reply:
Well, I personally use `nano`, I just figured most people use `vi`. I agree that this snippet is not super useful, so I'm okay with removing it. @johnhforrest, what do you think? Keep it or axe it?